packages/api-headless-cms/src/plugins/graphql.ts

```diff
@@ -54,6 +54,11 @@ export const createGraphQLPlugin = (): GraphQLSchemaPlugin => ({
 
             ${system.typeDefs}
         `,
+        /**
+         * TS is complaining about emptyResolver not receiving any arguments.
```

Comment:
There are a lot of explanations for `@ts-ignore`. When this PR is merged, everyone will need to go in the code and see how it can be improved. If I wenth through everything, it would take me months, probably.
Also, when I assigned `any`, I wrote comments for it (at least I tried to).